tests/testthat/test-fct_cross.R

```diff
@@ -0,0 +1,36 @@
+context("fct_cross")
+
+test_that("Gives correct levels", {
```

Comment:
I read "as test that gives correct levels", i.e. the label shouldn't start with a capital letter (and this file also needs restyling)

Reply:
Done